tools/commit-readme.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+#------------------------------------------------------------------------------
+# Commits the data files if any have changed
+#------------------------------------------------------------------------------
+
+if [ -z "$(git status --porcelain)" ]; then 
+	echo "Data did not change."
+else
+	echo "Data changed!"
+
+	# commit the result
+	git add README.md packages/**/README.md
```

Comment:
changed from `git add **/**/README.md` as it would reference `node_modules` as well